source/MaterialXView/ShaderValidators/Glsl/GlslProgram.cpp

```diff
@@ -23,7 +23,11 @@ GlslProgramPtr GlslProgram::creator()
 
 GlslProgram::GlslProgram() :
     _programId(UNDEFINED_OPENGL_RESOURCE_ID),
-    _hwShader(nullptr)
+    _hwShader(nullptr),
+    _indexBuffer(0),
```

Comment:
Number of changes here are related to moving the binding of hw resources out from the validator to here.